pkg/discovery/worker/k8s_discovery_worker.go

```diff
@@ -343,27 +347,19 @@ func (worker *k8sDiscoveryWorker) buildDTOs(currTask *task.Task) ([]*proto.Entit
 
 	// 3. build entityDTOs for containers
```

Comment:
nit: maybe we should update the comment to say "build entityDTOs for containers and containerSpecs"